auto_submit/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   googleapis: ^9.2.0
   googleapis_auth: ^1.3.1
   graphql: ^5.1.1
-  gql: ^0.13.1
+  gql: ^0.14.0
```

Comment:
(mentioned for line 11)
Do you intend to update sdk to `'>=2.19.0-0 <3.0.0'`?

Reply:
Yep! app_dart is intended to be built against the tip of tree, so this is an uprev to the latest version